Tests/appstoreconnect-cliTests/Operations/ReadPreReleaseVersionOperationTests.swift

```diff
@@ -0,0 +1,63 @@
+// Copyright 2020 Itty Bitty Apps Pty Ltd
+
+@testable import AppStoreConnectCLI
+import AppStoreConnect_Swift_SDK
+import Foundation
+import Combine
+import XCTest
+
+final class ReadPreReleaseVersionOperationTests: XCTestCase {
+    typealias Operation = ReadPreReleaseVersionOperation
+    typealias Options = Operation.Options
+
+    let successRequestor = OneEndpointTestRequestor(
+        response: { _ in Future({ $0(.success(dataResponse)) }) }
+    )
+
+    func testReturnsOnePreReleaseVersion() throws {
+        let operation = Operation(options: Options(filterAppId: "1504341572", filterVersion: "1.0"))
+        let output = try operation.execute(with: successRequestor).await()
+        XCTAssertEqual(output.preReleaseVersion.attributes?.version, "1.0")
+    }
+
```

Comment:
I guess we can do a step further by creating `noVersionRequestor` and `multipleVersionRequestor`, to test if we can get the correct error. 

We can also directly create a response model and pass to Requestor so we can get rid of the JSON data and have less code. I think testing the decoder doesn't bring much benefit.

Reply:
Added tests for noVersion and multiple versions.